### PR TITLE
Jitting cldprop from swrad

### PR DESCRIPTION
**Pull request by YakelynRJ**
This PR added @jit to mcica_subcol and cldprop, computing time is roughly the same given that jit takes ~13 seconds to compile.

 

Coverage reports (updated automatically):
- test_unit: [83%](https:\/\/output.circle-artifacts.com\/output\/job\/2d2ac22a-d32e-4322-bbfb-af71640fb891\/artifacts\/0\/tmp\/coverage\/htmlcov-test_unit\/index.html)

### CODE REVIEW

**Review by brianhenn — COMMENTED** [review]
@YakelynRJ looks good in terms of jitting the functions you describe. A couple minor cleanup things I noticed.

Can you change the name of this PR to be more accurate? `swrad` is itself is not jitted here. Also, is the next step to remove xarray inputs from `taumol` here and in LW?

File: external/radiation/python/radsw/radsw_main.py [hunk 1]
```diff
@@ -54,6 +54,7 @@
 from util import compare_data
 from config import *
 
+ngptsw = np.int(ngptsw)
```
[the remark this answers is not in the document]
Comment (YakelynRJ) [reply external/radiation/python/radsw/radsw_main.py:57]: Oh. I was testing if this was the problem. I forgot to remove them. I will remove this line.

File: external/radiation/python/radsw/radsw_main.py [hunk 2]
```diff
@@ -1238,6 +1265,33 @@ def swrad(
                     ipt,
                     ds_cldprtb,
```
[the remark this answers is not in the document]
Comment (YakelynRJ) [reply external/radiation/python/radsw/radsw_main.py]: I did not remove dataarrays from radsw_main yet. These changes will require many lines of change so I can remove them for the next PR.

File: external/radiation/python/radsw/radsw_main.py [hunk 3]
```diff
@@ -835,1046 +835,1050 @@ def mcica_subcol(iovrsw, cldf, nlay, ipseed, dz, de_lgth, ipt,rand2d):
...
+        cdat2,
+        cdat3,
+        cdat4,
+        cf1,
+        nlay,
+        ipseed,
+        dz,
+        delgth,
+        ipt,
+        ds,
```
[the remark this answers is not in the document]
Comment (YakelynRJ) [reply external/radiation/python/radsw/radsw_main.py]: Same. Is is still needed for now. I can remove in the next PR.

**Comment by YakelynRJ** [comment]
@brianhenn we realized that it won't be possible to jit taumol. That is why I didn't remove dataarrays from swrad yet.

File: external/radiation/python/radsw/radsw_main.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (brianhenn) [reply external/radiation/python/radsw/radsw_main.py]: Sorry, I was reviewing on a commit-by-commit basis and didn't see that you did remove `ds` in a later commit. Disregard

File: external/radiation/python/radsw/radsw_main.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (brianhenn) [reply external/radiation/python/radsw/radsw_main.py]: Similarly I was referring to `ds_cldprtb` since that wasn't removed until a later commit. Disregard

**Review by brianhenn — APPROVED** [review]
Thanks LGTM.

**Pull request merged**